.travis.yml

```diff
@@ -2,6 +2,8 @@ language: node_js
 node_js:
   - '4'
   - 'lts/*'
-  - 'node'
-env:
-  - PGUSER=postgres
+matrix:
+  include:
+    - name: 'Code Coverage'
+      node_js: 'stable'
+      script: 'npx nyc --reporter=lcov npm test && npx codecov'
```

Comment:
Can you add these as devDependencies and then provide an `npm coverage` script rather than using `npx` here?

Reply:
Sure for including the dev dependencies but how about leaving the command itself in travis.yml (without `npx`)?I don't see a point to including the `codecov` execution in an npm script as nobody would run it locally outside a CI environment.

Instead we add a `npm run coverage` script that runs coverage and opens your browser to view the results: `nyc --reporter=html npm test && open-cli coverage/index.html`

Reply:
Agreed—I'd leave out npx here. A `coverage` and `coverage-ci` npm script would be nice.